modules/reference/pages/kube-apis/enterprise-contract.adoc

```diff
@@ -2,7 +2,7 @@
 :anchor_prefix: k8s-api
 
 [id="reference"]
-== Enterprise Contract API Reference
+== Conforma API Reference
```

Comment:
Since the CRD is still refering to EnterpriseContract, perhaps this:

```suggestion
== Conforma (formerly Enterprise Contract) API Reference
```

Reply:
updated, thanks!